.buildkite/scripts/steps/cloud/purge_deployments.ts

```diff
@@ -29,12 +29,21 @@ for (const deployment of prDeployments) {
     const lastCommit = pullRequest.commits.slice(-1)[0];
     const lastCommitTimestamp = new Date(lastCommit.committedDate).getTime() / 1000;
 
+    const persistDeployment = Boolean(
+      pullRequest.labels.filter((label: any) => label.name === 'ci:cloud-persist-deployment').length
+    );
+    if (persistDeployment) {
+      continue;
```

Comment:
I think we should still terminate when the PR is closed.

Reply:
Pushed 33ae44b00c51b608981389fe06785d3a3f6cc397